### ISSUE TITLE
Collections still archived after upgrading

### ISSUE THREAD

**Issue by JP-from-Postman** [a85/POSTMan-Chrome-Extension#6027]
Please read through the [guidelines](https://github.com/postmanlabs/postman-app-support#guidelines-for-reporting-issues) before creating a new issue.

**Describe the bug**
For some free teams with Collections that have been archived, upgrading to Postman Pro doesn't un-archive the affected Collections.

Also, because paid teams _shouldn't_ need access to the web dashboard page for un-archiving Collections, they are not able to access it/un-archive their Collections directly.

**To Reproduce**
(Exact process unknown)
1. Create free team
2. Cause Collections to become archived
3. Upgrade team to Pro
4. (See if Collections remain archived)

**Expected behavior**
Upgrading to any paid Postman plan automatically restores all archived Collections

**Comment by preethammavin** [comment]
This issue is resolved, do feel free to reopen it if it reoccurs again.

**Comment by chrisdeso** [comment]
@preethammavin New report of this behavior added

**Comment by cyrielle-hada** [comment]
Workaround for now: logging out and signing back in fixed the issue.

@preethammavin can we see why a simple sync doesn't bring back the archived collections automatically? 

Logs can be found internally

**Comment by sankalp0o** [comment]
We've identified the root cause and are working on a solution so that this doesn't occur. It's technically complex so taking some time. In the meanwhile, if you don't see archived collections on upgrading, please unblock yourselves by logging out and logging back in.

**Comment by DannyDainton** [comment]
@sankalp0o Are we able to close out this issue now?

**Comment by sankalp0o** [comment]
@DannyDainton yes, this shouldn't be happening now and the issue can be closed.
